pkg/api/validation/validation_test.go

```diff
@@ -336,6 +356,11 @@ func TestValidatePersistentVolumes(t *testing.T) {
 		},
 	}
 
+	err := utilfeature.DefaultFeatureGate.Set("BlockVolumeSupport=true")
```

Comment:
You should have two sets of tests:  The first set doesn't set the feature gate and verifies that you get an error if you try to set the VolumeMode or VolumeDevices.  The second set of tests will set the feature gate and verify valid and invalid configurations.

Reply:
We shouldn't enable the alpha feature in this suite.  Instead, we can have a test case here to validate you can't specify PV.volumeType.